ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -368,7 +368,7 @@ module('Acceptance | aws | configuration', function (hooks) {
         .doesNotExist('Access type section does not render for a community user');
       // check all the form fields are present
       await click(GENERAL.toggleGroup('Root config options'));
-      for (const key of expectedConfigKeys('aws-root-create')) {
```

Comment:
the value passed into `expectedConfigKeys` is changed here because of the above mentioned changed: I cleaned up this method to create generic arrays of keys that could come in camelized or not camelized.